bitnami/elasticsearch/values.yaml

```diff
@@ -450,7 +450,7 @@ master:
   masterOnly: true
   ## @param master.replicaCount Number of master-elegible replicas to deploy
   ##
-  replicaCount: 2
+  replicaCount: 1
```

Comment:
Please mention the `replicaCount` value update in the PR description

Reply:
I have just reversed this change